test/integration/conflict_resolution.js

```diff
@@ -192,6 +192,164 @@ describe('Conflict resolution', () => {
     })
   })
 
+  // FIXME: Move to helpers?
+  const fullSyncStartingFrom = async (sideName) => {
+    if (sideName === 'local') {
+      // FIXME: Initial scan is not the same as watched change
+      await helpers.local.scan()
+      await helpers.remote.pullChanges()
+    } else {
+      // FIXME: Test remote first cases
+      throw new Error('Not implemented yet: fullSyncStartingFrom("remote")')
+    }
+    await helpers.syncAll()
+
+    // Simulate client restart (always starting from local)
+    await helpers.local.scan()
+    await helpers.remote.pullChanges()
+    await helpers.syncAll()
+  }
+
+  const bothSides = (tree) => ({local: tree, remote: tree})
+
+  describe('merging local file then remote one', () => {
+    it('renames the remote file', async () => {
```

Comment:
We actually don't know which one is renamed with this test. Or did I miss something?

Reply:
You're right, one *may* know it actually works this way, but the test definitely not shows anything.

I'll change the test description to something like *it renames one of the files*.